js/cards.js

```diff
@@ -85,15 +96,41 @@ export async function defineCardComponent() {
       return `${CLOUDINARY_URL}/q_auto/${this.data["Rarity Folder"]}/${this.data["Filename"]}.png`;
     }
 
+    resetCard() {
+      card_count = 10;
+      this.back.classList.remove("hidden");
+      this.front.tabIndex = "-1";
+
+      addHover();
+    }
+
     flipCard() {
-      this.front.classList.toggle("hidden");
-      this.back.classList.toggle("hidden");
+
+      if(gacha_display_selection !== "gacha-grid" && previous_card_front !== undefined){
+        previous_card_front.removeEventListener("mouseover", addAnimation);
+        previous_card_front.removeEventListener("click", addAnimation);
+      }
+
+      this.classList.add("clicked");
+      this.holder.classList.add("flip");
+      previous_card = this;
```

Comment:
what is the purpose of `previous_card`? I don't see it used elsewhere in this file. Delete?

Reply:
This was a leftover bit from the previous code - have deleted this too now. :)